Kickstarter-iOS/ViewModels/AppDelegateViewModel.swift

```diff
@@ -311,7 +311,7 @@ AppDelegateViewModelOutputs {
           else { return .init(value: params) }
 
         return AppEnvironment.current.apiService.fetchCategory(param: categoryParam)
-          .delay(AppEnvironment.current.apiDelayInterval, onScheduler: AppEnvironment.current.scheduler)
+          .ksr_delay(AppEnvironment.current.apiDelayInterval, onScheduler: AppEnvironment.current.scheduler)
```

Comment:
We now have our own versions of `delay` and `debounce` (e.g. https://github.com/kickstarter/Kickstarter-ReactiveExtensions/pull/54/files#diff-34b9cf9a3e017896fcf4b3ae5ba0c309R14) that take a `DispatchTimeInterval`, which is an `enum` of `seconds`/`milliseconds`/`microseconds`/`nanoseconds` cases. i think it's a friendlier way of dealing with time.